src/command_modules/azure-cli-consumption/HISTORY.rst

```diff
@@ -4,7 +4,7 @@ Release History
 ===============
 0.2.2
 +++++
-* Added new commands for budgets, marketplace and pricesheet APIs
+* Added new commands for pricesheet API
```

Comment:
As far as I understand this still doesn't capture the scope of changes in the PR.  You added one new command and added functionality to existing commands.  That should be spelled out as specifically as possible so users don't have to hunt for the new features.

Reply:
sure. i'll make the change.